extensions/eclipse-che-theia-plugin-ext/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@eclipse-che/theia-plugin-ext",
+    "version": "6.15.0",
```

Comment:
I thought that the lifecycle of che-theia would have been different of che repository (different versions)
Else I don't see why it's a separate repository ?